client/index.html

```diff
@@ -356,25 +357,352 @@ <h1 class="title">Settings</h1>
 							<div class="col-sm-12">
 								<textarea class="input" name="userStyles" id="user-specified-css-input" placeholder="You can override any style with CSS here"></textarea>
 							</div>
-							<div class="col-sm-12">
-								<h2>About The Lounge</h2>
+						</div>
+					</div>
+				</div>
+
+				<div id="help" class="window">
```

Comment:
I see where you're going with this: a "Quick Help" modal just like GitHub (hit <kbd>?</kbd> outside an `input`). While I see value in this because modals thrive when maintaining previous context on the view, I think this would become necessary if/when the help page becomes more comprehensive. So in other words, I think it's too early for this right now.

Other things to look at is that modals don't usually play well on mobile, and that we currently have no modals in the app to get inspiration from.
As mentioned in the PR description, my only goal with this PR is to move the actual content from the website to the client to move forward with documentation rework, and I cannot add another task to my current endless todolist :)

Are you OK keeping this in its separate window for now and improve it over time?